package.json

```diff
@@ -7,11 +7,14 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^29.5.0",
+    "@types/lodash": "^4.14.192",
     "@types/node": "^18.15.11",
     "@types/react": "^18.0.31",
     "@types/react-dom": "^18.0.11",
     "axios": "^1.3.4",
+    "lodash.debounce": "^4.0.8",
     "react": "^18.2.0",
+    "react-debounce-input": "^3.3.0",
```

Comment:
Can we delete one of the debounce packages?

Reply:
I agree that react-debounce-input should be removed if not in use.

Reply:
I uninstalled "react-debounce-input".